ApollosProject/apollos-prototype
# Willow Required Onboarding Changes

## DESCRIPTION

### What does this PR do, or why is it needed?

This PR makes a number of changes to the Onboarding (all backward compatible) that increase the composability and flexibility of the existing Onboarding components. 

The changes made are as follows: 

- Added names to all important `styled` components, so they they can be overridden from the `theme.js` file.
- On screens that create their own `styled`  components, attach those `styled` components to the `component` they are used in, so they can be easily used by people extending the component.  
- Removed the `BackgroundImage` from Onboarding

### What design trade-offs/decisions were made?

Have not done a thorough updating of `styled` names across UI-kit in this PR

### How do I test this PR?
Open onboarding for the Apollos Church, make sure it looks exactly the same as before.

### What automated tests did you write?
Updated existing snapshots

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
